### PR TITLE
ELevate Wiki Docs Updates to Include CRB Dependency Information

### PR DESCRIPTION
**Pull request by cyclops26**
PR to address issue #477.  Includes updates to frequently encounter issues documentation and the documentation for ELevating from CentOS7 to AlmaLinux 8/9.

### CODE REVIEW

**Review by codyro — APPROVED** [review]

**Comment by bennyvasquez** [comment]
Thank you!

**Comment by bennyvasquez** [comment]
@sboldyreva you're the elevate docs maven for sure. I'd love your eyes on this! Especially because I think #484 might change some of this, too.

File: docs/elevate/ELevating-CentOS7-to-AlmaLinux-9.md [hunk 1]
```diff
@@ -222,6 +222,20 @@ After these preparations are completed, you can upgrade your AlmaLinux 8 machine
...
+  Error: 
+  Problem: package nss_db-2.34-100.el9_4.2.x86_64 from @System requires glibc(x86-64) = 2.34-100.el9_4.2, but none of the providers can be installed
+
+  cannot install both glibc-2.34-100.el9_4.2.alma.2.x86_64 from baseos and glibc-2.34-100.el9_4.2.x86_64 from @System
+  cannot install both glibc-2.34-100.el9_4.2.x86_64 from baseos and glibc-2.34-100.el9_4.2.alma.2.x86_64 from baseos
+  cannot install the best update candidate for package glibc-2.34-100.el9_4.2.x86_64
+  problem with installed package nss_db-2.34-100.el9_4.2.x86_64
+  ```
+  This issue can be resolved by enabling the CRB repository. Either via `crb enable` if the EPEL repository is already enabeld, or via `dnf config-manager --set-enabled crb`. After the CRB repository is enabled, the dnf cache can be cleared and dnf update should return to normal functionality.
+
```
Comment (sboldyreva) [inline docs/elevate/ELevating-CentOS7-to-AlmaLinux-9.md:238]: It seems CRB/PowerTools could be the most popular case, but a similar issue can also be caused by another repo absence. I suggest pointing this to users and redirecting them to the repos page to check enabling commands - https://wiki.almalinux.org/repos/AlmaLinux.html

**Comment by bennyvasquez** [comment]
Thanks, @sboldyreva! @cyclops26, do you mind updating your PR to incorporate the feedback from Sofia?

**Comment by cyclops26** [comment]
Apologies @bennyvasquez and @sboldyreva, it has been a long few weeks at work. @sboldyreva I believe I addressed your feedback as you were indicating. Please let me know if any other changes are needed.

**Comment by sboldyreva** [comment]
@cyclops26 @bennyvasquez thank you, looks good to me!

**Review by bennyvasquez — APPROVED** [review]

**Pull request merged**